DRAFT.md

```diff
@@ -1554,24 +1599,21 @@ such types are copy constructible. We make `polymorphic` unconditionally copy
 and move constructible. The authors do not envisage that this could be relaxed
 in a future version of the C++ standard.
 
-### Implicit conversions
+### Converting constructors
 
-We decided that there should be no implicit conversion of a value `T` to an
-`indirect<T>` or `polymorphic<T>`. An implicit conversion would require using
-the free store and memory allocation, which is best made explicit by the user.
+After reciving feedback from LEWG, converting constructors were added to
+support conversion from `T` to an `indirect<T>` or `polymorphic<T>`. Because
+these operation allocate they memory are marked explicit so it the intent is
+clear.
```

Comment:
I think this should say “marked explicit so that the intent to use them made clear” or something like that.